spec/lib/stemcell/launcher_spec.rb

```diff
@@ -0,0 +1,72 @@
+require 'spec_helper'
+
+class MockInstance
+  def initialize(id)
+    @id = id
+  end
+
+  def id
+    @id
+  end
+end
+
+class MockException < StandardError
+end
+
+describe Stemcell::Launcher do
+  let(:sleep) { mock("sleep")}
+  let(:launcher) {
+    opts = {}
+    Stemcell::Launcher::REQUIRED_OPTIONS.map { |k| opts[k] = "" }
+    launcher = Stemcell::Launcher.new(opts)
+    launcher
+  }
+  instances = (1..4).map { |id| MockInstance.new(id) }
+  instance_ids = instances.map { |i| i.id }
+
+  OPERATION = 'op'
```

Comment:
`instances`, `instance_ids`, and `OPERATION` should probably be set using `let` too. The latter can just be a variable, not a constant.